lib/netext/httpext/transport.go

```diff
@@ -164,8 +168,33 @@ func (t *transport) measureAndEmitMetrics(unfReq *unfinishedRequest) *finishedRe
 			tags["ip"] = ip
 		}
 	}
+	var failed float64
+	if t.responseCallback != nil {
+		var statusCode int
+		if unfReq.response != nil {
+			statusCode = unfReq.response.StatusCode
+		}
+		passed := t.responseCallback(statusCode)
+		if enabledTags.Has(stats.TagPassed) {
+			if passed {
+				failed = 0
```

Comment:
Currently the metric value depends on whether the `passed` tag is enabled or not. These should be decoupled so that the metric emits the correct value regardless.

Reply:
yeah .. I guess I should move this,  2 lines up, outside the check for the tag :thinking:

Reply:
fixed in 17e5cc2